shell/platform/windows/flutter_windows.cc

```diff
@@ -81,6 +81,16 @@ void FlutterDesktopViewControllerForceRedraw(
   controller->view->ForceRedraw();
 }
 
+#ifndef WINUWP
+
+FLUTTER_EXPORT void FlutterDesktopViewControllerEnableDirectComposition(
```

Comment:
I would rather this wasn't conditionalized but instead was a FlutterSetDesktopCompositionMode taking an enum with values None, DirectComposition, WinUIComposition returning a bool to indicate success